src/Umbraco.Web.UI.Client/src/packages/core/content/workspace/content-detail-workspace-base.ts

```diff
@@ -64,6 +64,7 @@ export interface UmbContentDetailWorkspaceContextArgs<
 	contentTypeDetailRepository: UmbDetailRepositoryConstructor<ContentTypeDetailModelType>;
 	contentValidationRepository?: ClassConstructor<UmbContentValidationRepository<DetailModelType>>;
 	skipValidationOnSubmit?: boolean;
+	ignorerValidationOnSubmit?: boolean;
```

Comment:
I think there might be some confusion about the difference between " skip " and " ignore " 🤔 Additionally, an extra "r" has sneaked into "ignore.".

Reply:
Yes, that is a good perspective.

But there is a difference, the intention here is not to be breaking.. so you can still choose to skipValidation.

And now we can then choose to do validation, but ignore the result of it.

Reply:
(i have corrected the r mistake)